referencia/webpayrest/README.md

```diff
@@ -79,38 +59,38 @@ el ambiente de integración](/documentacion/como_empezar#ambientes).
 ### Credenciales del Comercio
 
 ```java
-// Tbk-Api-Key-Id: Próximamente...
-// Tbk-Api-Key-Secret: Próximamente...
+// Tbk-Api-Key-Id: Código de comercio
+// Tbk-Api-Key-Secret: Secreto
```

Comment:
Why not use the code and secrete here?

Reply:
I find this section to be an explanation of what they are instead of how to configure